shap/utils/_legacy.py

```diff
@@ -37,7 +37,8 @@ def kmeans(X, k, round_values=True):
     imp = SimpleImputer(missing_values=np.nan, strategy='mean')
     X = imp.fit_transform(X)
 
-    kmeans = KMeans(n_clusters=k, random_state=0).fit(X)
+    # Specify `n_init` for consistent behaviour between sklearn versions
+    kmeans = KMeans(n_clusters=k, random_state=0, n_init=1).fit(X)
```

Comment:
I think the default used to be 10, e.g. a very old version of sklearn 0.24: https://scikit-learn.org/0.24/modules/generated/sklearn.cluster.k_means.html#sklearn.cluster.k_means

for compatibility, it might be safer to stick to 10.

Reply:
Done